block-storage/src/main/scala/coop/rchain/blockstorage/ApprovedStoreSyntax.scala

```diff
@@ -0,0 +1,31 @@
+package coop.rchain.blockstorage
+
+import cats.effect.Sync
+import cats.implicits._
+import coop.rchain.blockstorage.approvedStore.ApprovedStore
+import coop.rchain.casper.protocol.{ApprovedBlock, ApprovedBlockProto}
+import coop.rchain.shared.syntax._
+import scodec.bits.ByteVector
+
+trait ApprovedStoreSyntax {
+  implicit final def syntaxApprovedStore[F[_]: Sync](
+      approvedStore: ApprovedStore[F]
+  ): ApprovedStoreOps[F] =
+    new ApprovedStoreOps[F](approvedStore)
+}
+
+final class ApprovedStoreOps[F[_]: Sync](
+    // ApprovedStore extensions / syntax
+    private val approvedStore: ApprovedStore[F]
+) {
+  val approvedBlockKey: Byte = 42.toByte
+
+  def getApprovedBlock: F[Option[ApprovedBlock]] = approvedStore.get1(approvedBlockKey)
+
+  def putApprovedBlock(block: ApprovedBlock): F[Unit] = approvedStore.put(approvedBlockKey, block)
+
+  /**
+    * This is fatal error from which Approved Store can not be recovered.
+    */
+  case class ApprovedStoreFatalError(message: String) extends Exception(message)
```

Comment:
This is left over.

Reply:
Yes, you are right. This is my carelessness. I [removed](https://github.com/rchain/rchain/pull/3655/commits/bda8ca1f0b83e68a48c75cda0af64c2e5bcb637c) it.